starboard/android/shared/audio_renderer_passthrough.cc

```diff
@@ -138,6 +138,11 @@ void AudioRendererPassthrough::WriteSamples(const InputBuffers& input_buffers) {
   decoder_->Decode(
       input_buffers,
       std::bind(&AudioRendererPassthrough::OnDecoderConsumed, this));
+  // AC-3/EAC-3 sync frames with 1536 samples at 48khz have a length of 32 ms.
+  // TODO: Determine this value at runtime.
+  const SbTime kBufferLength = 32 * kSbTimeMillisecond;
```

Comment:
nit: kBufferDuration

Also consider moving the call to CacheMultipleDiscardDurations() before the call to decoder_->Decode() to avoid potential race conditions.